packages/tailwindcss/src/candidate.test.ts

```diff
@@ -616,6 +623,13 @@ it('should parse a utility with an arbitrary value with parens including a typeh
   `)
 })
 
+it('should not parse a utility with an arbitrary value with parens including a typehint that does not start with --', () => {
+  let utilities = new Utilities()
+  utilities.functional('bg', () => [])
+
+  expect(run('bg-(color:my-color)', { utilities })).toMatchInlineSnapshot(`[]`)
+})
+
```

Comment:
We should add tests for stuff like bg-foo/(bar) to make sure it's thrown out too

Reply:
Ah good call!